native_client/BUILD

```diff
@@ -8,14 +8,27 @@ load("@build_bazel_rules_swift//swift:swift.bzl", "swift_c_module", "swift_libra
 load("@emsdk//emscripten_toolchain:wasm_rules.bzl", "wasm_cc_binary")
 
 
+config_setting(
```

Comment:
why? this is already defined by tensorflow

Reply:
sorry, I think this was a merge conflict not well resolved. Removed.